docs/python/tutorial-django.md

```diff
@@ -678,14 +678,16 @@ For example, add the following class in `models.py` to define a data model that
 
 ```python
 from django.db import models
+from django.utils import timezone
 
 class LogMessage(models.Model):
     message = models.CharField(max_length=300)
     log_date = models.DateTimeField("date logged")
 
     def __str__(self):
         """Returns a string representation of a message."""
-        return f"'{self.message}' logged on {self.log_date.strftime('%A, %d %B, %Y at %X')}"
+        date = timezone.localtime(self.log_date)
+        return f"'{self.message}' logged on {date.strftime('%A, %d %B, %Y at %X')}
```

Comment:
@kraigb Are you missing a trailing double quote at the end of the return statement

Reply:
I checked the example repo and added the trailing double quote to match.

Reply:
Thanks. I hadn't noticed that the quote disappeared...wonder why that happened...but thanks for fixing it and the sample!